### PR TITLE
Add ability to ignore files, add new keywords.

### PR DESCRIPTION
**Pull request by Skeen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,28 @@ Django Template i18n lint
 [![PyPI version](https://pypip.in/v/django-template-i18n-lint/badge.png)](https://pypi.python.org/pypi/django-template-i18n-lint)
 [![PyPI Downloads](https://pypip.in/d/django-template-i18n-lint/badge.png)](https://pypi.python.org/pypi/django-template-i18n-lint)
 
+Fork of original project by Rory McCann, [https://github.com/rory/django-template-i18n-lint](https://github.com/rory/django-template-i18n-lint), description by original author: [Lint tool to find non-i18n strings in a django template](http://www.technomancy.org/python/django-template-i18n-lint/)
```
Comment (jayvdb) [inline README.md:9]: these are the same repo.  I think you should remove

File: README.md [hunk 2]
```diff
@@ -6,16 +6,28 @@ Django Template i18n lint
...
+
 
 A simple script to find non-i18n text in a Django template.
 
-It can also automatically wrap the strings in `{% trans "" %}` tags, by running it with the `-r` command-line flag.
```
Comment (jayvdb) [inline README.md:12]: I think the `-r` mode is worth mentioning on the README.

File: README.md [hunk 3]
```diff
@@ -6,16 +6,28 @@ Django Template i18n lint
...
+    $ python django_angular_template_i18n_lint.py template_files
+
+Program docs are available:
 
-For more info see [Lint tool to find non-i18n strings in a django template](http://www.technomancy.org/python/django-template-i18n-lint/)
+    $ python django_angular_template_i18n_lint --h
 
-Code is copyright Rory McCann 2013, and dual licenced under the GNU GPL version3 (or at your option a later version), and the BSD licence. See the files LICENCE.GPLv3 and LICENCE.BSD for more information
 
+Usefull hints:
```
Comment (jayvdb) [inline README.md:26]: one `l` in useful

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by Skeen** [comment]
I do not utilize this project anymore, and thus I am unfortunately not gonna work to get this merged.

**Pull request closed without merging**